apps/remote_control/lib/lexical/remote_control/search/store/mnesia/connection.ex

```diff
@@ -0,0 +1,206 @@
+defmodule Lexical.RemoteControl.Search.Store.Mnesia.Connection do
```

Comment:
At the risk of nesting overload, maybe this should live under `State` to better indicate that it shouldn't be used directly (it's delegated to in `State`).

```suggestion
defmodule Lexical.RemoteControl.Search.Store.Mnesia.State.Connection do
```

Reply:
worse, I want to move `Mnesia` to `backends` along with `Ets`